import/controls/TextSelectionController.qml

```diff
@@ -158,8 +160,20 @@ MouseArea {
         }
     }
 
+    function selectionRect() {
+        var selectionX = Math.min(start.x, end.x)
+        var selectionY = Math.min(start.y, end.y)
+        var selectionWidth = Math.max(start.x, end.x) - selectionX + end.width
+        var selectionHeight = Math.max(start.y, end.y) - selectionY + end.height
+        var rv = Qt.rect(selectionX, selectionY, selectionWidth, selectionHeight)
+        return rv
```

Comment:
Not quite sure why but initial size is off after entering to the text selection mode. After moving a bit left or right handle, selection rectangle updates to somewhat meaningful.